apps/roam/src/components/BirdEatsBugs.tsx

```diff
@@ -195,7 +195,7 @@ export const initFeedbackWidget = (
       },
       ui: {
         position: "bottom-right",
-        defaultButton: { icon: undefined },
+        defaultButton: false, // hide, will be triggered in DiscourseFloatingMenu
```

Comment:
<img width="1112" height="382" alt="image" src="https://github.com/user-attachments/assets/f4e0f458-e3e8-4530-8cb4-c5075442382a" />

If false should be an option, let's update the type definition.

Reply:
Ah, did not realize type definition was here, I thought it came from the module. 🤦‍♂️